fighthealthinsurance/common_view_logic.py

```
@@ -1026,6 +1051,7 @@ def create_or_update_denial(
         primary_professional: Optional[ProfessionalUser] = None,
         patient_user: Optional[PatientUser] = None,
         patient_visible: bool = False,
+        subscribe: bool = False, # Note: we don't handle this
```

Ruff (0.14.6)

Unused class method argument: `subscribe`

(ARG003)